packages/base-types/src/node/visual.ts

```diff
@@ -62,6 +62,9 @@ export interface ImageStepData extends BaseStepData {
   frameType?: FrameType;
   dimensions: Nullable<Dimensions>;
   canvasVisibility: CanvasVisibility;
+  options: {
+    loop: Nullable<boolean>;
```

Comment:
is it possible we could change this to just `loop: boolean`? it's much cleaner to just default to `false` instead of dealing with 3 possible values (`null | true | false`)